examples/square/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "square"
+version = "0.9.0"
+edition = { workspace = true }
+rust-version = { workspace = true }
+repository = { workspace = true }
+
+[dependencies]
+anyhow = { workspace = true }
+serde = { workspace = true, features = ["derive"] }
+serde_json =  { workspace = true }
+wit-bindgen-rt = { version = "0.32.0", features = ["bitflags"] }
+
+[lib]
+crate-type = ["cdylib"]
+
+[profile.release]
+codegen-units = 1
+opt-level = "s"
+debug = false
+strip = true
+lto = true
+
+[package.metadata.component]
```

Comment:
Note sure where to comment, as you won't see the other file, but please remove bindings.rs and add that to .gitignore